src/avfoundation.cs

```diff
@@ -6415,10 +6418,12 @@ interface AVMutableMovieTrack_AVMutableMovieTrackTrackAssociations
 	interface AVFragmentedMovieTrack
 	{
 		[Mac (10, 10)]
+		[Notification]
 		[Field ("AVFragmentedMovieTrackTimeRangeDidChangeNotification")]
 		NSString ATimeRangeDidChangeNotification { get; }
```

Comment:
misnamed field, i.e. `A` prefix - and easy to miss since it won't be seen as a typo, so better correct it now so the new API are named correctly